server/src/main/java/org/elasticsearch/index/shard/IndexShardOperationPermits.java

```diff
@@ -269,6 +305,28 @@ int getActiveOperationsCount() {
         }
     }
 
+    /**
+     * @return a list of containing an exceptions for each permit that wasn't released yet. The stack traces of the exceptions
+     *         was captured when the operation acquired the permit and their message contain the debug information supplied at the time.
+     */
+    List<RuntimeException> getActiveOperations() {
```

Comment:
can you test this method in IndexShardOperationPermitsTests that it returns something meaningful?

Reply:
yeah, so I was doubting about that one. What is meaningful? I can check there is something for each open op and also the message contains the debug info. Is that what you mean?

Reply:
yes. To check that the active operations are actually captured.